v2/apiserver/internal/authx/named_principals.go

```diff
@@ -12,40 +12,16 @@ var (
 // root is an implementation of the Principal interface for the "root" user.
 type root struct{}
 
-func (r *root) Roles() []Role {
-	return []Role{
-		RoleAdmin(),
-		RoleEventCreator(RoleScopeGlobal),
-		RoleProjectAdmin(RoleScopeGlobal),
-		RoleProjectCreator(),
-		RoleProjectDeveloper(RoleScopeGlobal),
-		RoleProjectUser(RoleScopeGlobal),
-		RoleReader(),
-	}
-}
-
```

Comment:
All these named principals were deleted for now and will eventually reappear elsewhere. This wasn't the right place for them.